#Next waypoint
float32 dest_lat
float32 dest_long
int16 waypoint_id
float32 l_motor_val
float32 r_motor_val
bool relay_state